tests/console/test_console_udevrule.py

```diff
@@ -15,7 +15,8 @@ def test_console_port_mapping(duthost):
     ttys = set(out.split())
     pytest_assert(len(ttys) > 0, "No console tty devices been created by udev rule")
 
-    for i in range(1, 49):
+    out = duthost.console_facts()["ansible_facts"]["console_facts"]["lines"].keys()
+    for i in range(1, len(out)+1):
```

Comment:
For C0-* device, the number should aligned with the console line number in configuration.
Let's enumerate all keys here, and expect C0-{key} exists.